reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Variables.md

```diff
@@ -93,14 +93,53 @@ $Splat = @{ Name = "Win*"; Include = "WinRM" }
 Invoke-Command -Session $s -ScriptBlock { Get-Service @Using:Splat }
 ```
 
-## Using local variables in PowerShell 2.0
+### Other situations where the 'Using' scope modifier is needed
+
+For anything that executes out-of-runspace, doesn't execute directly in the
+caller's runspace, you need the `Using` scope modifier in order to embed
+variable values from the caller's scope, so that the out-of-runspace code
+can access it. (Conversely, all other contexts neither require nor support the
+'Using' scope modifier.) Specifically, this includes the following contexts:
+
+- Remotely executed commands, started with Invoke-Command's `-ComputerName` or
```

Comment:
Again, I feel we should not use 'runspace':

...(remote session)
...(out-of-process session)
...(separate thread session)